07-uxdi-collaboration/ajax/readme.md

```diff
@@ -120,31 +104,36 @@ But Andy, i thought we were able to access that big json object we saw before? W
 
 We can drill through this response just like any other JS object.
 
-```javascript
+```js
+// Note that we use traditional Javascript object notation to access the response.
 .done(function(response)(){
   console.log(response.current_observation.temp_f)
 })
 ```
 
-> This is huge. We now have a way to access information from an API as a JSON object. With it we can do a myriad of things.
+## YOU DO: DOM Manipulation Using Response Data (10/60)
 
-## You do - (10/60)
-- Take our existing code for the the weather underground app. Instead of logging the temperature, the `.done()` promise should create a div with the current wind speed in mph.
+Take our existing code for the the weather underground app. Instead of logging the temperature, the `.done()` promise should create a div with the current wind speed in mph.  
 
-### BONUS!
-- Create an Input text field for City and State in the HTML
-- Have the endpoint url change dynamically based on user input to generate a div about current weather in that area.
+### Bonus
+  1. Create an input text field for City and State in the HTML.  
+  2. Have the endpoint url change dynamically based on user input to generate a div about current weather in that area.  
 
-## Break - 10/70
+## Break (10/70)
 
-## AJAX - CUD intro (5/75)
-So we've used AJAX to do an asynchronous `get` request to a third party API. But it wouldn't make sense for us to be able to do CUD functionality to that same site. They probably don't want anyone that's not a developer there to be able to update the weather however we want. That is not to say that kind of functionality doesn't exist, we just don't have access to it.
+## Intro: AJAX and CRUD (5/75)
+So we've used AJAX to do an asynchronous `GET` request to a 3rd party API. But it wouldn't make sense for us to be able to do CRUD functionality to that same site. They probably don't want anyone that's not a developer there to be able to update the weather however we want. That is not to say that kind of functionality doesn't exist, we just don't have access to it.
```

Comment:
what is the purpose of this intro? maybe a subheader could be something like "combining ajax and internal APIs". and why you would even do that